src/xgov-dapp/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "algorand-voting-tool",
   "private": true,
-  "version": "0.0.0",
+  "version": "2.1.0",
```

Comment:
@HashMapsData2Value sorry for the late review, but the code is actually testing VERSION of `dapp` and not `xgov-dapp` so this should be fixed